src/components/Button/Button.tsx

```diff
@@ -69,7 +69,8 @@ export class Button extends React.Component<IButtonProps, IButtonState> {
           'aria-label': ariaLabel,
           'aria-labelledby': ariaLabel ? null : labelId,
           'aria-describedby': ariaDescription ? ariaDescriptionId : description ? descriptionId : null,
-          'ref': (c: HTMLButtonElement): HTMLButtonElement => this._buttonElement = c
+          'ref': (c: HTMLButtonElement): HTMLButtonElement => this._buttonElement = c,
+          'href' : href ? href : null
```

Comment:
this may generate a warning when rendering as a button, which doesn't have an href.

If it does, a quick workaround is to add it as a separate object in the assign call:
assign(
{},
rootProps,
(href ? { href } : null,
{ other stuff }
)